### PR TITLE
fix(examples): add cloudflared support

### PR DESCRIPTION
**Pull request by jokesterfr**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Cloudflared, named `mytun` internally @PrestaShop SA, is a edge tunneling solution, such as ngrok (already supported within PrestaShop Flashlight). The goal is to quickly allow any developer to boostrap a PrestaShop environment and expose it online. It is specially indicated when testing PrestaShop cloud solutions such as ps_accounts.
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixing #89
| Sponsor company   | PrestaShop SA
| How to test?      | `cd ./examples/cloudflared-tunnel && open README.md`

### CODE REVIEW

File: examples/basic-example/docker-compose.yml [hunk 1]
```diff
@@ -10,6 +10,8 @@ services:
       # - DEBUG_MODE=true
     ports:
       - 8000:80
+    expose:
```
Comment (Venatum) [inline examples/basic-example/docker-compose.yml:13]: We use expose only for this compose ?

File: examples/cloudflared-tunnel/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+name: flashlight-ngrok-tunnel
```
Comment (Venatum) [inline examples/cloudflared-tunnel/docker-compose.yml:1]:
```suggestion
name: flashlight-cloudflared-tunnel
```

File: examples/cloudflared-tunnel/README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+Cloudflare implement 2 ways of tunneling traffic from its edge ip to your local PrestaShop instance:
+
+1. Quick Tunnels
+2. Named Tunnels
+
+Only the Named tunnels (2) are supported in this example for now. Feel free to contribute to automatically guess the quick tunnels metrics from PrestaShop Flashlight.
+
+# Setting up
+
+First you have to setup the credentials files:
```
Comment (Venatum) [inline examples/cloudflared-tunnel/README.md:12]:
```suggestion
First, you have to set up the credential files:
```

File: examples/cloudflared-tunnel/README.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
+# Example: cloudflared tunnel
+
+Cloudflare implement 2 ways of tunneling traffic from its edge ip to your local PrestaShop instance:
```
Comment (Venatum) [inline examples/cloudflared-tunnel/README.md:3]:
```suggestion
Cloudflare implements two ways of tunneling traffic from its edge ip to your local PrestaShop instance:
```

**Review by Venatum — APPROVED** [review]

File: examples/basic-example/docker-compose.yml (hunk 1 above)
Comment (jokesterfr) [reply examples/basic-example/docker-compose.yml:13]: `expose` is only for the sake of documentation. I'm not sure if it is more useful than confusing for a Docker user 🤔

**Pull request merged**